cloud/openstack/os_server.py

```diff
@@ -360,13 +361,18 @@ def _create_server(module, cloud):
             metas[k] = v
         module.params['meta'] = metas
 
+    if type(module.params['security_groups']) is str:
+        module.params['security_groups'] = module.params['security_groups'].split(',')
```

Comment:
I'm just trying to stick with the existing code style in this module.  I agree totally, but I'd like to make compartmentalized changes.